crypto/ml_kem/ml_kem_ipd_ref_common/params.c

```diff
@@ -0,0 +1,41 @@
+#include <openssl/base.h>
+
+#include "params.h"
+
+static void ml_kem_params_init(ml_kem_params *params, size_t k) {
+  assert((k == 2) || (k == 3) || (k == 4));
```

Comment:
release builds may ignore this `assert`, leading to unexpected data for the below conditionals.